Core/src/org/sleuthkit/autopsy/mainui/datamodel/events/DAOEventUtils.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2021 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.mainui.datamodel.events;
+
+import java.beans.PropertyChangeEvent;
+import org.sleuthkit.autopsy.ingest.IngestManager;
+import org.sleuthkit.autopsy.ingest.ModuleContentEvent;
+import org.sleuthkit.autopsy.ingest.ModuleDataEvent;
+import org.sleuthkit.datamodel.AbstractFile;
+import org.sleuthkit.datamodel.Content;
+
+/**
+ *
+ * Utilities for handling events in DAO
+ */
+public class DAOEventUtils {
+
+    /**
+     * Returns the content from the event. If the event does not
+     * contain a event or the event does not contain Content, null
+     * is returned.
+     *
+     * @param evt The event
+     *
+     * @return The inner content or null if no content.
+     */
+    public static Content getContentFromEvt(PropertyChangeEvent evt) {
```

Comment:
I think this and getDerivedContentFromEvt() need a new name because they seem to only be relevant to certain types of events.  I.e. if you pass in a DATA_ADDED event, then it won't return anything. Even though the DataArtifact is content. Maybe something like getContentFromFileEvent() - it's not great, but gives some indication that it's type-specific.